dwitter/templates/base.html

```diff
@@ -23,13 +23,45 @@
     {% load staticfiles %}
     <link href="https://fonts.googleapis.com/css?family=Roboto:400,400i,700,700i" rel="stylesheet">
     {% compress css %}
-    <link rel=stylesheet href='{% static "main.css" %}' >
+    <link id='style' rel=stylesheet href='{% static "main.css" %}' >
     {% endcompress %}
 
     {% block head %}
     {% endblock %}
   </head>
   <body class="{% block body_class %}{% endblock %}">
+    <script type="text/javascript">
+      var switchTheme = function(){
+        // Change the imported stylesheet from main.css to main_dark.css or vice versa
+        // Change localstorage darkMode to true/false
+        var mode = localStorage.getItem('darkMode');
+        if (mode == 'true')
```

Comment:
Instead of `darkMode = 'true'` or `'false'` you could use `colorScheme = 'dark'` or `'default'`, or even simpler: `'dark'` or `''`

This would leave room for additional colourschemes in future, for example in case someone produces a "high-contrast" theme for those people who need it.

Reply:
Done :)... Used 'theme' with the settings you suggested, as per my response to lionleaf...